docs/en/quantize.md

```diff
@@ -1,11 +1,77 @@
-[English](../en/quantize.md) | 简体中文
+[简体中文](../cn/quantize.md) ｜English 
 
-# 量化加速
+# Quantization Acceleration
 
-简要介绍量化加速的原理。
+Quantization is a popular method in model compression to obtain models with smaller sizes and faster inference speeds.
+Based on PaddleSlim, FastDeploy integrates a one-click model quantization tool. FastDeploy supports the deployment of quantized models for faster inference.
 
-目前量化支持在哪些硬件及后端的使用
+## FastDeploy supports quantitative model deployment for multiple engines and hardware
```

Comment:
FastDeploy supports quantized model deployment using multiple inference backends and hardware.